### ISSUE TITLE
Select widget not displaying options

### ISSUE THREAD

**Issue by dust-io** [netlify/netlify-cms#2980]
The issue is that the select widget isn't displaying options — this is happening on a website I'm working on but also on the demo here:

https://cms-demo.netlify.com/#/collections/kitchenSink/entries/a-big-entry-with-all-the-things

It has been working fine but suddenly stopped — I've double checked the admin config.yml numerous times and it is valid YAML.

The issue is in multiple browsers.

netlify-cms-app 2.10.0
netlify-cms-core 2.13.0
netlify-cms 2.10.0

 Git provider: Github
 OS: Mac
 Browser version: Chrome Version 79.0.3945.79 (Official Build) (64-bit)

**CMS configuration**

```
# Everything you need to know about configuring the CMS can be found here:-
# https://www.netlifycms.org/docs/configuration-options/

backend:
  name: git-gateway # Netlify’s Git Gateway connects to Git provider’s API
  branch: master # Branch to update (master by default)

media_folder: "assets/uploads" # Folder where user uploaded files should go

collections: # A list of collections the CMS should be able to edit
  # POSTS
  # - name: "members" # Used in routes, ie.: /admin/collections/:slug/edit
  #   identifier_field: member_name
  #   label: "Member" # Used in the UI, ie.: "New Post"
  #   folder: "_members" # The path to the folder where the documents are stored
  #   sort: "title:asc" # Default is title:asc
  #   create: true # Allow users to create new documents in this collection
  #   fields: # The fields each document in this collection have
  #   - { label: "Name", name: "member_name", widget: "string" }
  #   - { label: "Description", name: "member_desc", widget: "text" }
  #   - { label: "Website", name: "member_web", widget: "string", required: false }
  - name: "projects" # Used in routes, ie.: /admin/collections/:slug/edit
    label: "Project" # Used in the UI, ie.: "New Post"
    folder: "_projects" # The path to the folder where the documents are stored
    sort: "title:asc" # Default is title:asc
    create: true # Allow users to create new documents in this collection
    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
    editor:
      preview: false
    fields: # The fields each document in this collection have
      - { label: "Layout", name: "layout", widget: "hidden", default: "project" }
      - { label: "Featured Image", name: "featured_image", widget: "image" }
      - { label: "Title", name: "title", widget: "string" }
      - { label: "Client", name: "client", widget: "string" }
      - { label: "Collaborators", name: "collaborators", widget: "list", required: false }
      - label: "Categories"
        name: "categories"
        widget: "select"
        multiple: true
        required: false
        options:
          - { label: "Publication", value: "Publication" }
          - { label: "Brand", value: "Brand" }
          - { label: "Illustration", value: "Illustration" }
          - { label: "Type", value: "Type" }
          - { label: "Website", value: "Website" }
          - { label: "Exhibition", value: "Exhibition" }
          - { label: "Record", value: "Record" }
          - { label: "Promotion", value: "Promotion" }
      - label: "Key Words"
        name: "tags"
        widget: "select"
        multiple: true
        required: false
        options: 
          - { label: "Artist", value: "Artist" }
          - { label: "Authorial", value: "Authorial" }
          - { label: "Book Cover", value: "Book Cover" }
          - { label: "Community", value: "Community" }
          - { label: "Contemporary Art", value: "Contemporary Art" }
          - { label: "Craft", value: "Craft" }
          - { label: "Creative Writing", value: "Creative Writing" }
          - { label: "Design", value: "Design" }
          - { label: "Documentary", value: "Documentary" }
          - { label: "Education", value: "Education" }
          - { label: "Environment", value: "Environment" }
          - { label: "Event", value: "Event" }
          - { label: "Exhibition", value: "Exhibition" }
          - { label: "Festival", value: "Festival" }
          - { label: "Food", value: "Food" }
          - { label: "Food & Drink", value: "Food & Drink" }
          - { label: "Football", value: "Football" }
          - { label: "Gum", value: "Gum" }
          - { label: "Health", value: "Health" }
          - { label: "History", value: "History" }
          - { label: "Journal", value: "Journal" }
          - { label: "Lead", value: "Lead" }
          - { label: "Music", value: "Music" }
          - { label: "Packaging", value: "Packaging" }
          - { label: "Performance", value: "Performance" }
          - { label: "Photography", value: "Photography" }
          - { label: "Play", value: "Play" }
          - { label: "Research", value: "Research" }
          - { label: "Residency", value: "Residency" }
          - { label: "Screenprint", value: "Screenprint" }
          - { label: "Sheffield", value: "Sheffield" }
          - { label: "Theatre", value: "Theatre" }
          - { label: "Venue", value: "Venue" }
          - { label: "Writing", value: "Writing" }
      - label: "Year"
        name: "year"
        widget: "select"
        multiple: false
        required: false
        options: 
          - { label: "1999", value: "1999" }
          - { label: "2000", value: "2000" }
          - { label: "2001", value: "2001" }
          - { label: "2002", value: "2002" }
          - { label: "2003", value: "2003" }
          - { label: "2004", value: "2004" }
          - { label: "2005", value: "2005" }
          - { label: "2006", value: "2006" }
          - { label: "2007", value: "2007" }
          - { label: "2008", value: "2008" }
          - { label: "2009", value: "2009" }
          - { label: "2010", value: "2010" }
          - { label: "2011", value: "2011" }
          - { label: "2012", value: "2012" }
          - { label: "2013", value: "2013" }
          - { label: "2014", value: "2014" }
          - { label: "2015", value: "2015" }
          - { label: "2016", value: "2016" }
          - { label: "2017", value: "2017" }
          - { label: "2018", value: "2018" }
          - { label: "2019", value: "2019" }
          - { label: "2020", value: "2020" }
      - label: "Year End"
        name: "year_end" 
        widget: "select"
        multiple: false
        required: false
        options: 
          - { label: "1999", value: "1999" }
          - { label: "2000", value: "2000" }
          - { label: "2001", value: "2001" }
          - { label: "2002", value: "2002" }
          - { label: "2003", value: "2003" }
          - { label: "2004", value: "2004" }
          - { label: "2005", value: "2005" }
          - { label: "2006", value: "2006" }
          - { label: "2007", value: "2007" }
          - { label: "2008", value: "2008" }
          - { label: "2009", value: "2009" }
          - { label: "2010", value: "2010" }
          - { label: "2011", value: "2011" }
          - { label: "2012", value: "2012" }
          - { label: "2013", value: "2013" }
          - { label: "2014", value: "2014" }
          - { label: "2015", value: "2015" }
          - { label: "2016", value: "2016" }
          - { label: "2017", value: "2017" }
          - { label: "2018", value: "2018" }
          - { label: "2019", value: "2019" }
          - { label: "2020", value: "2020" }
          - { label: "Onwards", value: "Onwards" }
      - { label: "Description", name: "description", widget: "markdown", required: false }
      - label: "Images"
        name: "images"
        widget: "list"
        types:
        - label: "Full Width Image"
          name: "full_width_image"
          widget: object
          fields:
            - label: "Full Width Image"
              name: "full_image"
              widget: image
              media_library:
                config:
                  multiple: false
        - label: "Two Image Row"
          name: "two_image_row"
          widget: object
          fields:
            - label: "Left Image"
              name: "left_image"
              widget: image
              required: false
              media_library:
                config:
                  multiple: false
            - label: "Right Image"
              name: "right_image"
              widget: image
              required: false
              media_library:
                config:
                  multiple: false
        
    # PAGES
  - name: "home"
    identifier_field: name
    label: "Home"
    folder: "home"
    sort: "title:asc"
    create: false
    slug: "{{slug}}"
    editor:
      preview: false
    fields:
      - {label: "Layout", name: "layout", widget: "hidden", default: "home"}
      - {label: "Title", name: "title", widget: "string"}
      - {label: "Permalink", name: "permalink", widget: "hidden"}
      - {label: "Section", name: "section", widget: "hidden", default: "{{name}}"}
      - {label: "Writing", name: "home_writing", widget: "markdown", required: true}
  - name: "credits"
    identifier_field: name
    label: "Credits"
    folder: "credits"
    sort: "title:asc"
    create: false
    slug: "{{slug}}"
    editor:
      preview: false
    fields:
      - {label: "Layout", name: "layout", widget: "hidden", default: "credits"}
      - {label: "Title", name: "title", widget: "string"}
      - {label: "Permalink", name: "permalink", widget: "hidden"}
      - {label: "Section", name: "section", widget: "hidden", default: "{{name}}"}
      - {label: "Writing", name: "credits_writing", widget: "markdown", required: true}
      - label: "Studio Members"
        name: "studio_members"
        label_singular: "Member"
        widget: "list"
        fields:
          - {label: "Name", name: "member_name", widget: "string", default: "Mitch"}
          - {label: "Description", name: "member_desc", widget: "text" }
          - {label: "Website", name: "member_web", widget: "string", default: "http://marston.com"}
```

**Comment by railconcept** [comment]
same issue with select widget and relation widget it does'nt show options anymore !!!!!!!

**Comment by MWDelaney** [comment]
Can a release be tagged for this? It's causing some issues with production sites for me.

**Comment by railconcept** [comment]
Still having the same issue  select widget and relation widget it doesn't show options anymore !!!!!!!

**Comment by rwboyer** [comment]
I would also like some sort of way to fix this on production sites that were working via the 2.1 netlify cms that is running most of my sites right now. Anyone figure out how to do that?

Thanks

**Comment by erquhart** [comment]
This didn’t get released properly, looking into it.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by erquhart** [comment]
Fixed in netlify-cms@2.10.2-beta.0 and netlify-cms@2.10.2.
